src/corelibs/U2Formats/src/PDBFormat.h

```diff
@@ -99,6 +103,9 @@ class U2FORMATS_EXPORT PDBFormat : public TextDocumentFormatDeprecated {
         bool seqResContains(char chainIdentier, int residueIndex, char acronym);
         QByteArray getNextSpecLine();
 
+        // Returns the end-of-name index for `specification`.
+        int endOfNameInd(const QString &specification);
```

Comment:
The method is not a simple getter but changes the internal parser state.
Please update the doc and change the name accordingly.

Reply:
The new title looks awful, but the conversation is resolved.